docs/DeveloperGuide.md

```diff
@@ -294,17 +294,71 @@ The activity diagram shows the general sequence of steps when a user interacts w
 
 <br>
 
+## Implementation of Entity Commands
+Entity commands include `add`, `delete`, `find`, `clear` commands
+
+**Step 1**. The user types an `xyz` command in the `CommandBox`, followed by the type of entity `person` or `appt`, followed by appropriate arguments and prefixes.
+**Step 2**. The command is passed to the `LogicManager`. `LogicManager` then calls the `AddressBookParser::parseCommand` method to parse the command
```

Comment:
Just a minor typo, there seems to be missing a full stop at the end of the sentence.